tests/test_header.py

```diff
@@ -223,6 +223,102 @@ def test_find_and_replace_keep_shebang():
     """When encountering a shebang, keep it and put the REUSE header beneath
     it.
     """
+    spdx_info = SpdxInfo(
+        set(["GPL-3.0-or-later"]), set(["SPDX" "-FileCopyrightText: Jane Doe"])
+    )
+    text = cleandoc(
+        """
+        #!/usr/bin/env python3
+
+        # This is a test file. Hello.
+
+        pass
+        """
+    )
+    expected = cleandoc(
+        """
+        #!/usr/bin/env python3
+
+        # spdx-FileCopyrightText: Jane Doe
+        #
+        # spdx-License-Identifier: GPL-3.0-or-later
+
+        # This is a test file. Hello.
+
+        pass
+        """
+    ).replace("spdx", "SPDX")
+
+    assert find_and_replace_header(text, spdx_info) == expected
+
+
+def test_find_and_replace_keep_shebang_spdx():
+    """When encountering a shebang, keep it and put the REUSE header beneath
```

Comment:
This comment is identical between all tests. It probably shouldn't be.

Reply:
Ooops.